server.go

```diff
@@ -95,13 +100,16 @@ func NewFromConfig(conf Config) (ret Server, err error) {
 
 	interval, err := time.ParseDuration(conf.Interval)
 	if err != nil {
+		logrus.Fatalf("Error parsing configuration %s", err)
```

Comment:
Previously this would return the error to the caller, which seems to be what is done elsewhere in this function. It seems like it would be nice to be consistent about it, and either (a) return any errors to the caller, or (b) don't ever return an error, and crash/log Fatal?